packages/es-client/indexer.js

```diff
@@ -372,6 +411,33 @@ function deletePdr({
   });
 }
 
+/**
+ * Deletes the async operation in ElasticSearch
+ *
+ * @param  {Object} params
+ * @param  {Object} params.esClient - ElasticSearch Connection object
+ * @param  {string} params.id - the async operation ID
+ * @param  {string[]} [params.ignore] - Array of response codes to ignore
+ * @param  {string} params.index - Elasticsearch index alias (default defined in search.js)
+ * @param  {string} params.type - Elasticsearch type (default: asyncOperation)
+ * @returns {Promise} Elasticsearch response
+ */
+function deleteAsyncOperation({
```

Comment:
Is there a reason why we don't need to write a unit test for this function?

Reply:
These functions are just shallow calls directly through to `deleteRecord`, but I'll add some unit tests